src/escpos/escpos.py

```diff
@@ -733,6 +738,18 @@ def panel_buttons(self, enable=True):
         else:
             self._raw(PANEL_BUTTON_OFF)
 
+    def queryStatus(self):
+        self._raw(RT_STATUS_ONLINE)
+        time.sleep(1)
+        status = self._read();
+        if (len(status) > 0):
+            return status;
+        else:
+            return [8];
+
+    def isOnline(self):
+        status = self.queryStatus()[0];
+        return ((status & RT_MASK_ONLINE) == 0);
```

Comment:
All changes have been pushed but the & -> and one. If I'm not wrong, the operator must be '&' since a binary 'and' is needed.